app/bundles/CoreBundle/Command/PurgeStaleNotificationsCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Command;
+
+use DateTime;
+use Doctrine\ORM\EntityManager;
+use Mautic\CoreBundle\Entity\Notification;
+use Mautic\CoreBundle\Entity\NotificationRepository;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class PurgeStaleNotificationsCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $inputOptions = [
+            new InputOption(
+                'stale-days',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Notificiations from "X" days ago will be considered stale.',
+                '-7 day'
+            ),
+            new InputOption(
+                'dry-run',
+                null,
+                InputOption::VALUE_NONE,
+                'Do a dry run without actually deleting anything.'
+            ),
+        ];
+
+        $this->setName('mautic:notifications:purge')
+            ->setDescription("Purge stale users' notfications.")
+            ->setDefinition($inputOptions)
+            ->setHelp(<<<'EOT'
+The <info>%command.name%</info> command is used to purge stale user's notifications
+
+<info>php %command.full_name%</info>
+
+You can optionally set the --stale-days flag to consider what is a stale notification:
```

Comment:
This reads better to me.
```suggestion
You can set the --stale-days flag to change the number of days to consider stale:
```